app/controllers/auth0_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Auth0Controller < ApplicationController
+  def callback
+    # This stores all the user information that came from Auth0
+    # and the IdP
+    session[:userinfo] = request.env["omniauth.auth"]
+
+    # Redirect to the URL you want after successful auth
+    redirect_to "/dashboard"
```

Comment:
Do you think we should use the Rails URL helpers where possible?

```suggestion
    redirect_to dashboard_path
```